environments/staging/public.yml

```diff
@@ -36,6 +36,10 @@ couch_dbs:
 
 DROPBOX_APP_NAME: 'CommCareHQFiles - Staging'
 
+s3_blob_db_enabled: 'true'
```

Comment:
I think this should just be `true` (without the quotes)

Reply:
Yeah I wasn't totally sure about this, but the line in `all.yml` is

```
s3_blob_db_enabled: "{{ 'true' if groups.get('riakcs') else '' }}"
```

Maybe I'm lapsing on my ansible syntax knowledge; my impression is that this would leave this value with one of the _strings_ `'true'` or `''`. Rather than digging into it, I just followed the pattern. Looking into it now, it seems like that value is only ever used for its truthiness, so

```
s3_blob_db_enabled: "{{ True if groups.get('riakcs') else False }}"
```

should do just as well, and then I can change this line to

```
s3_blob_db_enabled: yes
```
like you suggested

Reply:
I thought there were cases where true/false values got converted to a string (`'True'` or `'False'`), making them both truthy in future evaluations. Using `'true'` and `''` evades this since any non-empty string is truthy and an empty string is falsey. You can ignore my input here if we know the issue with Ansible has been fixed or if I'm just mis-remembering (e.g., if no one else has ever observed that issue in this context).

Reply:
Interesting. I think that in this context they do evaluate correctly to booleans. The test I came up with was to run `cchq swiss update-config --branch=dmr/riakcs-to-aws` (in just check mode). Since there it's supposed to be set to `False`, if it's not getting set correctly it'll be interpreted as truthy and then that would change some values in localsettings.py; when I ran it I didn't see any related changes, so I'm interpreting that as implying we're safe here.